presto-main/src/main/java/com/facebook/presto/operator/scalar/annotations/ParametricScalarImplementation.java

```diff
@@ -248,11 +369,16 @@ public MethodHandle getMethodHandle()
         private final Optional<MethodHandle> constructorMethodHandle;
         private final List<ImplementationDependency> constructorDependencies = new ArrayList<>();
         private final List<LongVariableConstraint> longVariableConstraints;
+        private final Class<?> returnNativeContainerType;
+        private NullConventionFlag nullConventionFlag;
+
+        private final List<ParametricScalarImplementationChoice> choices = new ArrayList<ParametricScalarImplementationChoice>();
 
         private Parser(String functionName, Method method, Optional<Constructor<?>> constructor)
         {
             this.functionName = requireNonNull(functionName, "functionName is null");
             this.nullable = method.getAnnotation(SqlNullable.class) != null;
+            ParametricScalarImplementation.NullConventionFlag nullConventionFlag = nullable ? NullConventionFlag.NULLABLE_ARGUMENT : NullConventionFlag.NOT_NULLABLE_ARGUMENT;
```

Comment:
Here `NullConventionFlag` is indicating the return convention, which is not a natural usage.